editor/src/core/shared/element-template.ts

```diff
@@ -1077,7 +1119,7 @@ export function jsxConditionalExpression(
     type: 'JSX_CONDITIONAL_EXPRESSION',
     uid: uid,
     originalConditionString: originalConditionString,
-    condition: condition,
+    condition: fixJSXConditionalExpressionCondition(condition, comments), // remove any duplicate comments shared with the element
```

Comment:
Is this optimisation necessary? It seems like this runs the risk of deleting user comments which might have been deliberately duplicated (e.g. `eslint-disable-next-line`)

Reply:
this is only removing duplicate `@utopia/` flags that were passed over between the condition and the conditional element, because of the way comments are parsed. It was an existing issue btw, but it was not surfacing. Fixing this lazily is the best approach I could come up with compared with the alternative of fixing the parsing, making it _much_ more complex

Reply:
Ok in that case I'm fine with it, as long as it's not going to cause issues with a chain of overrides of the form ```// @utopia/conditional=true```